service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionController.java

```diff
@@ -164,7 +162,7 @@ public void handleSubmittedEvent(
         @RequestBody @NotNull CallbackRequest callbackRequest) {
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
-        if (featureToggleService.isPaymentsEnabled()) {
+        if (featureToggleService.isPaymentsEnabled() && displayAmountToPay(caseDetails)) {
```

Comment:
Thanks for taking care of this onoe as well.

Reply:
This should resolve the purple banner stuff right?